submissions/Jigsaff/a-tiny-JS-world-post-OOP/index.js

```diff
@@ -19,37 +9,37 @@ class Organism {
         this.friends = [];
     }
 
-    addFriends(...Friends) {
-        Friends.forEach(({name}) => this.friends.push(name));
+    addFriends(...friends) {
+        friends.map(({name}) => this.friends.push(name));
         return this.friends;
     }
 
     toString() {
         let strings = [
             `type: ${this.type}; `,
-            `legs: ${this.legs}; `,
-            `hands: ${this.hands}; `,
             `name: ${this.name}; `,
             `gender: ${this.gender}; `,
+            `legs: ${this.legs}; `,
+            `hands: ${this.hands}; `,
```

Comment:
If there is no hands, maybe try to not print this ?) This is not blocker, but will be cool if you implement this feature)